src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -428,3 +428,38 @@ def get_system_routing_stack():
         raise OSError("Cannot detect routing stack")
 
     return result
+
+# Check if System warm reboot or Container warm restart is enabled.
+def is_warm_restart_enabled(container_name):
+    state_db = SonicV2Connector(host='127.0.0.1')
+    state_db.connect(state_db.STATE_DB, False)
+
+    TABLE_NAME_SEPARATOR = '|'
+    prefix = 'WARM_RESTART_ENABLE_TABLE' + TABLE_NAME_SEPARATOR
+
+    # Get the system warm reboot enable state
+    _hash = '{}{}'.format(prefix, 'system')
+    wr_system_state = state_db.get(state_db.STATE_DB, _hash, "enable")
+    wr_enable_state = True if wr_system_state == "true" else False
+
+    # Get the container warm reboot enable state
+    _hash = '{}{}'.format(prefix, container_name)
+    wr_container_state = state_db.get(state_db.STATE_DB, _hash, "enable")
+    wr_enable_state |= True if wr_container_state == "true" else False
+
+    state_db.close(state_db.STATE_DB)
+    return wr_enable_state
+
+# Check if System fast reboot is enabled.
+def is_fast_reboot_enabled():
+    fb_system_state = 0
+    cmd = 'sonic-db-cli STATE_DB get "FAST_REBOOT|system"'
```

Comment:
Maybe add a TODO or FIXME comment here as a reminder to clean this up once the entry is changed to a hash value.

Reply:
@jleveque  I have added this code reference in the **issue** which Ying raised to track this cleanup item (https://github.com/Azure/sonic-buildimage/issues/5690#issuecomment-714258980). 
Is that ok ?

Reply:
That's fine. Thanks.